homeassistant/components/mqtt/cover.py

```diff
@@ -396,6 +396,9 @@ def supported_features(self):
         if self._config.get(CONF_COMMAND_TOPIC) is not None:
             supported_features = OPEN_CLOSE_FEATURES
 
+        if self._config.get(CONF_PAYLOAD_STOP) != "":
```

Comment:
This should compare against `None` and you should update the validation to allow `None` to be passed in .

Reply:
Doing it the other way around would make more sense. Remove the default value because it's not a required option of covers. However, that would be too much of a breaking change.

Reply:
That's how I went initially and then reverted because it wasn't necessary to add a breaking change for this feature

Same for the none, just wasn't sure how to pass `None` in yaml, quick search got me
`payload_stop:`

I'll clean it up and squash the commits

Reply:
Done